ARCCore/src/main/com/andoverrobotics/core/config/Configuration.java

```diff
@@ -0,0 +1,213 @@
+package com.andoverrobotics.core.config;
+
+import java.io.File;
+import java.io.FileReader;
+import java.io.IOException;
+import java.io.Reader;
+import java.lang.reflect.Field;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Objects;
+import java.util.Properties;
+
+/**
+ * A key-value mapping abstraction that allows for the retrieval of values in a variety of types.
+ * <p> <h2>Schemas</h2> Configuration Schemas are small classes that only contain non-private fields
```

Comment:
@jzhu2020 Here is the documentation you were looking for.

Reply:
Alright, thanks.